supplementary_style_guide/glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc

```diff
@@ -0,0 +1,762 @@
+
+[discrete]
+[[fuse]]
+==== image:images/caution.png[with caution] FUSE (File system in USEr Space)
+*Description*: A software interface for Linux and Linux-like systems that lets non-privileged users create and configure their own file systems without interacting directly with kernel code.
+
+*Use it*: With caution - it is more typical to refer to FUSE-compatible or FUSE-compliant systems rather than FUSE itself.
+
+[discrete]
+[[geo-replication]]
+==== image:images/caution.png[with caution] geo-replication (noun)
+*Description*: In Red Hat Gluster Storage and related solutions, geo-replication refers to asynchronous replication of data in a Gluster file system from one location to another across a network.
+
+In Red Hat Quay, geo-replication refers to replication of data from one central storage engine to other storage engines.
+
+*Use it*: With caution - ensure no cross-over of the above definitions.
+
+[discrete]
+[[gluster-volume]]
+==== image:images/yes.png[yes] gluster volume (noun)
+*Description*: A gluster volume is a logical collection of bricks across one or more servers in a trusted storage pool.
+
+*Use it*: Yes
+
+[discrete]
+[[hard-disk-drive]]
+==== image:images/yes.png[yes] hard disk drive (noun)
+*Description*: In physical storage media, the hard disk drive is a physical component that stores data and all of the electronics that support or drive the reading and writing of data to and from the storage. It may be inserted into or removed from a server.
+
+*Use it*: Yes
+
+*Incorrect forms*: disk drive
+
+[discrete]
+[[hyperconverged-cluster]]
+==== image:images/yes.png[yes] hyperconverged cluster (noun)
+*Description*: A generic term to refer to the set of physical machines providing compute and storage capabilities in a Red Hat Hyperconverged Infrastructure for Virtualization (RHHI-V) or Red Hat Hyperconverged Infrastructure for Cloud (RHHI-C) cluster.
+
+*Use it*: Yes
+
+*Incorrect forms*: hyper converged cluster, hyper-converged cluster
+
+
+[discrete]
+[[infrastructure-node]]
+==== image:images/yes.png[yes] infrastructure node (noun)
+*Description*: A node that is labeled to run pieces of the OpenShift Container Platform environment, as distinct from a node intended to run applications.
+
+*Use it*: Yes
+
+
+[discrete]
+[[initiator]]
+==== image:images/yes.png[yes] initiator (noun)
+*Description*: The consumer of storage from a target. The initiator is typically a server with an adapter card. The initiator “initiates” a connection over the network fabric to one or more ports on your storage system, which are called target ports.
+
+*Use it*: Yes
+
+[discrete]
+[[internal-mode]]
+==== image:images/yes.png[yes] internal mode (noun)
+*Description*: Using internally hosted services, for example, Red Hat OpenShift Container Storage where all components of OpenShift Container Storage are deployed within a cluster managed by OpenShift Container Platform and benefit from operator-based deployment and management.
```

Comment:
operator -> Operator (is always capitalized per OpenShift)

Reply:
Fixed, thankyou